notebooks/config.ipynb

```diff
@@ -0,0 +1,1046 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "d023d283-7e1d-4e75-95b8-5033bea42a59",
+   "metadata": {},
+   "source": [
+    "# Config Tool\n",
+    "\n",
+    "The `uwtools` API's `config` module provides functions to create and manipulate configuration files, objects, and dictionaries.\n",
+    "\n",
+    "For more information, please see the <a href=\"https://uwtools.readthedocs.io/en/main/sections/user_guide/api/config.html\">uwtools.api.config</a> Read the Docs page.\n",
```

Comment:
I think it might be a good idea to pin to the `stable` version over the `main` version of the docs. My thought was that `stable` will present a version of our most recent release, where `main` will pull directly from the latest on our `main` branch in Git. I'm not sure if there's a preference for this. I'd say if I was a user, I'd probably want something in a release rather than in development. Thoughts from the rest of the team?

https://uwtools.readthedocs.io/en/stable/sections/user_guide/api/config.html

Reply:
I think this is probably a good idea in this case and in most cases but am curious to hear other's thoughts as well. There may be times where it would be nice to add a feature to the notebooks that exists on `main` but not in a release yet (the file -> fs change comes to mind) where we might need to initially link the `main` version of the docs and change it later.

Reply:
I went ahead and made this change for this notebook and the template notebook too.

Reply:
Good questions. One option might be to pin to a _specific_ release (not even just the latest release) that we are sure the notebooks are compatible with. Then, when we release a new version, we could intentionally revisit the notebooks, run their tests but also read through them to make sure they still make sense: It's one thing to pass automated code tests, it's another to ensure that the cell commentary still actually describes the code cells.

If we pinned to `stable` (which, if I'm not mistaken, is essentially what we're doing with the `uwtools` package itself in `binder/environment.yml` by not specifying a specific version number) then, when `notebooks` is integrated into the `main` branch, our CI tests should at least ensure that the code correct via the unit tests. But we'd still have to have the discipline to go review the prose part of the notebooks.

I think I'd prefer to pin both docs and the package to a specific release, and then schedule occasional work to update those pins and read through the notebooks to make sure they're still correct.

It would be good to hear from others, and ultimately from PO @christinaholtNOAA.

Reply:
I agree with Paul's assessment. Let's pin to a specific release and revisit the notebooks occasionally. Let's make sure that the version is very clear to the user of the notebook.

Reply:
I updated this notebook and the template notebook to mention a specific release and link to the same Read the Docs version of the page, but left the fs notebook alone since there isn't a numbered release that has the file -> fs change yet.